src/backend/base/langflow/api/v2/mcp.py

```
@@ -101,27 +101,29 @@
     session: DbSession,
     storage_service=Depends(get_storage_service),
     settings_service=Depends(get_settings_service),
+    actionCount: bool = False,
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)